modules/db2/src/main/java/org/testcontainers/containers/Db2Container.java

```diff
@@ -0,0 +1,94 @@
+package org.testcontainers.containers;
+
+import com.github.dockerjava.api.command.InspectContainerResponse;
+import org.testcontainers.DockerClientFactory;
+import org.testcontainers.containers.output.OutputFrame;
+import org.testcontainers.containers.output.WaitingConsumer;
+import org.testcontainers.utility.LogUtils;
+
+import java.io.IOException;
+import java.util.concurrent.TimeoutException;
+import java.util.function.Predicate;
+
+public class Db2Container<SELF extends Db2Container<SELF>> extends JdbcDatabaseContainer<SELF> {
```

Comment:
No need for self-typing here
```suggestion
public class Db2Container extends JdbcDatabaseContainer<Db2Container> {
```